common/extensions/api/_permission_features.json

```diff
@@ -20,5 +20,19 @@
       "226CF815E39A363090A1E547D53063472B8279FA",  // Media Router Release
       "A321D47A2B4CA86898167A55CA8B2E02385EA7CD"   // Brave Shields
     ]
+  }],
+  "ipfs": [{
+    "channel": "trunk",
+    "extension_types": ["extension", "platform_app"]
+  },
+  {
+    "channel": "stable",
+    "extension_types": ["extension"],
+    "whitelist": [
```

Comment:
could you please add comments indicating what each of these extensions is?

Reply:
done